code/modules/atmospherics/machinery/atmosmachinery.dm

```diff
@@ -407,6 +436,12 @@
 		if(trinary == TRUE && (piping_layer == 1 || piping_layer == 5))
 			PIPING_FORWARD_SHIFT(pipe_overlay, piping_layer, 2)
 
+/obj/machinery/atmospherics/proc/pipe_overlay(set_icon, set_state, direction, color = COLOR_VERY_LIGHT_GRAY, piping_layer = 3, set_layer = PIPE_VISIBLE_LEVEL)
+	var/image/pipe_overlay
+	pipe_overlay = . = image(icon = set_icon, icon_state = set_state, layer = set_layer, dir = direction)
```

Comment:
```suggestion
	pipe_overlay = image(icon = set_icon, icon_state = set_state, layer = set_layer, dir = direction)
```

Then `return pipe_overlay` at the end. This is needlessly confusing.

Reply:
done, next commit